File: testdata/io/diff-v2.ts
import fs from 'node:fs';

export function syncData() {
  fs.readFileSync('in.txt', 'utf8');
  fetch('https://example.com/items');
  fs.writeFileSync('out.txt', 'next');
}

class Writer {
  Save() {
    fs.writeFileSync('out.txt', 'ok');
  }
}

void Writer;

File: testdata/dart/io/diff-v2.dart
import 'dart:io';

Future<void> syncData() async {
  await File('input.txt').readAsString();
  print('saved');
}

class ApiClient {
  Future<void> status() async {
    print('ok');
    await http.get(Uri.parse('https://example.com/items'));
  }
}
